### PR TITLE
custom network

### PR DESCRIPTION
**Pull request by vird**
Adds capability to connect and send transactions to arweave.localtest, arweave.testnet and other testnet identifiers

### CODE REVIEW

**Comment by rosmcmahon** [comment]
hi @vird sorry no one has taken a look at this until now. do you still need this? maybe you could merge current master with this patch, and ensure tests are passing?

**Review by rosmcmahon — COMMENTED** [review]
can you merge ArweaveTeam:master into this?

[edit: sorry I just saw that I already asked this]

File: src/common/lib/api.ts [hunk 1]
```diff
@@ -38,6 +39,7 @@ export default class Api {
       timeout: config.timeout || 20000,
       logging: config.logging || false,
       logger: config.logger || console.log,
+      network: config.network,
```
Comment (rosmcmahon) [inline src/common/lib/api.ts:42]: we should probably give a default value here? like `"arweave.N.1"` - is that right?

Comment (vird) [reply src/common/lib/api.ts:42]: Default value is undefined. Do not send any header if not specified

**Pull request merged**